commands/operator-sdk/cmd/up/local.go

```diff
@@ -201,10 +201,11 @@ func upLocalHelm() {
 	for gvk, factory := range factories {
 		// Register the controller with the factory.
 		err := controller.Add(mgr, controller.WatchOptions{
-			Namespace:       namespace,
-			GVK:             gvk,
-			ManagerFactory:  factory,
-			ReconcilePeriod: helmOperatorFlags.ReconcilePeriod,
+			Namespace:               namespace,
+			GVK:                     gvk,
+			ManagerFactory:          factory,
+			ReconcilePeriod:         helmOperatorFlags.ReconcilePeriod,
+			WatchDependentResources: true,
```

Comment:
Should watching dependent resources be configurable, or is this always being `true` fine for `up local` purposes?

Reply:
It probably makes sense to make this configurable in both the `up local` and in-cluster scenarios. 

Since this is also a feature of the ansible operator, how about we do a follow-up PR to add a flag for this to the `pkg/internal/flags.WatchFlags` struct?

@shawn-hurley, any input from the ansible operator perspective (or otherwise)?